site/guide/model-validation/view-reports.qmd

```diff
@@ -5,77 +5,121 @@ filters:
 
-How reports are organized:
+Reports are primarily designed for model validators or Chief Risk Officers (CROs) who require an understanding of findings related to model validation activities.
+
+::: {.prereq}
+
+## Prerequisites
+
+- [x] {{< var link.login >}}
+- [x] There are models registered in the model inventory.[^1]
+
+:::
+
+## Reports overview
+
+Reports are broken down by data on models and data on findings: 
 
 :::: {.flex .flex-wrap .justify-around}
 
 ::: {.w-50-ns}
 
-- **Findings by risk area** — Identifies issues based on the type of risk they pose to the organization, such as model limitations, data quality issues, regulatory compliance, and more. Helps in understanding the risk landscape for your models and risk areas requiring attention or improvement.
-
-- **Findings by business unit** — Segmented according to different divisions or departments within your organization, such as Finance, Marketing, and more. Assists you in pinpointing where in the organization model-related issues are occurring, facilitating more targeted resolution, if necessary.
+![Reports page showing several bar charts with model data](reports-models.png){fig-alt="Screenshot of the Reports page that shows several bar charts with model data"}
 
 :::
 
 ::: {.w-50-ns}
 
-- **Findings by status** — Sorts findings based on their resolution status, such as open, closed, or past due. Provides a snapshot of the validation process, highlighting which issues have been addressed, which are currently being worked on, and which are yet to be tackled.
-
-- **Findings by model** — Specific to individual models, details the findings for each model, including any errors, limitations, or assumptions that may affect the model's performance. This helps in assessing the reliability and accuracy of each model.
+![Reports page showing several bar charts with findings data](reports-findings.png){fig-alt="Screenshot of the Reports page that shows several bar charts with findings data"}
 
 :::
 
 ::::
 
-::: {.prereq}
+::: {.callout}
+Reports will not include data or findings from archived models.[^2]
+:::
 
-## Prerequisites
+#### Reports on models
 
-::: {.callout title="Before you begin"}
-Make sure you are logged in to the [{{< var validmind.platform >}} {{< fa hand-point-right >}}](/guide/configuration/log-in-to-validmind.qmd).
-:::
+model summary
+: Count of your total models registered with ValidMind.
 
-Reports are primarily designed for model validators or Chief Risk Officers (CROs) who require an understanding of findings related to model validation activities.
+number of models by business unit and tier
+: Count of your model volume by business unit[^3] further broken down by risk tier. Assists you in pinpointing where in the organization resources should be concentrated on, along with the materiality of risk that unit may pose based on the models they are responsible for. 
 
-:::
+number of models by status and business unit
```

Comment:
```suggestion
Number of models by status and business unit
```

Here's the example you requested. These entries look like they refer to UI elements that are part of the report. They should match the casing of the UI, or be init-capped. As is, they look like glossary entries, which they are not.

Reply:
Ah, yes, OK, i see what you mean. I think I did that because some casing needed to be fixed in the UI, which I will wrap into the next time I stick my fingers in frontend.